cleanrepo/CleanRepo/Program.cs

```diff
@@ -33,7 +33,8 @@ static void Main(string[] args)
         //   }
         //
         // ... to avoid hardcoded values in DEBUG preprocessor directives like this:
-        args = new[] { "--orphaned-articles", "--articles-directory=../../../../../../testrepo2/docs", "--delete=false" };
+        args = new[] { "--relative-links", "--articles-directory=c:\\users\\gewarren\\docs\\docs", "url-base-path=/dotnet" };
```

Comment:
Is this OK?

Reply:
Yes, this is only used in the debug version.